sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -290,6 +302,57 @@ public virtual async Task<Response> RevokeTokensAsync(CommunicationUserIdentifie
             }
         }
 
+        /// <summary>Issues a token for a <see cref="CommunicationUserIdentifier"/>.</summary>
+        /// <param name="communicationUser">The <see cref="CommunicationUserIdentifier"/> for whom to issue a token.</param>
+        /// <param name="cancellationToken">The cancellation token to use.</param>
+        /// <exception cref="RequestFailedException">The server returned an error.</exception>
+        public virtual Response<CommunicationTurnCredentialsResponse> IssueTurnCredentials(CommunicationUserIdentifier communicationUser, CancellationToken cancellationToken = default)
+        {
+            var RestClientForTurn = new CommunicationIdentityRestClient(
```

Comment:
Why not use the `RestClient` that is instantiated in the constructor? It should not be instantiating it every time.

Reply:
I see this as temporary until the Rest endpoint version matches that of the identity service. Currently both are at different versions, hence need separate clients.

Reply:
Maybe we should have two members `RestClient` and `TurnRestClient` both created in the constructor. Let's make sure the new preview api also support all the `2021-03-07` routes as this looks hacky.

Reply:
Updated.